features/steps/user_initialisation_steps.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=E0102, E0602
```

Comment:
`features/steps/user_initialisation_steps.py:58:0: E0102: function already defined line 11 (function-redefined)` - well that's just how `behave` works.

`Undefined variable 'when/then/given' (undefined-variable)` - I dunno whether this has to do with the import. And that import was the devs' instruction to begin with, so plz no h8 😿  I'm telling mom otherwise

Reply:
The latter got replaced with E0611, since pylint can't seem to find those decorators in behave package, somehow.